metrics_utility/base/collector.py

```diff
@@ -289,9 +289,26 @@ def _gather_csv_collections(self):
 
         last_key = None
 
+        disable_job_host_summary_str = os.environ.get('METRICS_UTILITY_DISABLE_JOB_HOST_SUMMARY_COLLECTOR', 'false')
+        disable_job_host_summary = disable_job_host_summary_str.lower() == 'true'
+
+        optional_collectors = get_optional_collectors()
+
         for collection in self.collections[Collection.COLLECTION_TYPE_CSV]:
             if last_key != collection.key:
-                logger.warning(f'Progress info: Now gathering {collection.key}')
+                write_enabled = False
+
+                if collection.key == 'job_host_summary' and not disable_job_host_summary:
+                    write_enabled = True
+
+                if collection.key in optional_collectors:
+                    write_enabled = True
+
+                if write_enabled:
+                    logger.warning(f'Progress info: Now gathering {collection.key}')
+                else:
+                    logger.warning(f'Progress info: Skipping {collection.key} because it is not enabled.')
```

Comment:
```suggestion
                if write_enabled:
                    logger.debug(f'Progress info: Now gathering {collection.key}')
                else:
                    logger.debug(f'Progress info: Skipping {collection.key} because it is not enabled.')
```

(unless you intentionally want to list every skipped collector on every collect?)

Reply:
This should be maybe info, but previously, it was part of standard output without --verbose.

It prints every collector only once, but current change is informing user that this collector is skipped.

Reply:
Aah, makes sense :+1: :)